src/components/Events/SingleEvent.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+
+  <div class="event-container">
+    <router-view></router-view>
+      <div class="event-name">
```

Comment:
You may be able to do this cleaner using css flexbox as opposed to grid. Then you wouldn't need to have all of these css classes and divs. For example, it would be a 2x4 grid with the bottom left and top right being flex boxs with the flex direction down. So the top row would just 33% of the grid and the bottom row would be the rest. Then you would only need to wrap the top right and bottom right in divs.
![image](https://user-images.githubusercontent.com/44480372/73412589-0fe09600-42d7-11ea-9786-b25cb0978210.png)

Reply:
I tried making these changes, let me know if this was what you had in mind or if I misunderstood your comment!